PTGLgraphComputation/src/plcc/Main.java

```diff
@@ -5940,7 +5940,9 @@ public static MolContactInfo calculateAtomContactsBetweenResidues(Molecule a, Mo
                         contact[bIntID][aIntID][0][statAtomIDj]++;
                     } catch(java.lang.ArrayIndexOutOfBoundsException e) {
                         //DP.getInstance().w("calculateAtomContactsBetweenResidues():Contact statistics array out of bounds. Residues with excessive number of atoms detected: " + e.getMessage() + ".");
-                        DP.getInstance().w("calculateAtomContactsBetweenResidues(): Atom count for residues too high (" + e.getMessage() + "), ignoring contacts for these atoms (aIntID=" + aIntID + ", bIntID=" + bIntID + ", statAtomIDi=" + statAtomIDi + ", statAtomIDj=" + statAtomIDj + ").");
+                        if (! Settings.getBoolean("PTGLgraphComputation_B_no_warn")) {
+                            DP.getInstance().w("calculateAtomContactsBetweenResidues(): Atom count for residues too high (" + e.getMessage() + "), ignoring contacts for these atoms (aIntID=" + aIntID + ", bIntID=" + bIntID + ", statAtomIDi=" + statAtomIDi + ", statAtomIDj=" + statAtomIDj + ").");
+                        }
```

Comment:
This should not be handled here but in DP's function w(arning), so that not each time we want to pass a warning have to surround it with this check. As a matter of fact, DP's both w(arning)s already respect the setting. So in my opinion, these lines should not affect the behavior of the program. If they do, please investigate why.